### PR TITLE
Add standard logging

### PR DESCRIPTION
**Pull request by meluru**

### CODE REVIEW

File: indico/Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ RUN ${INDICO_VIRTUALENV}/indico setup create_symlinks /opt/indico
 RUN ${INDICO_VIRTUALENV}/indico setup create_logging_config /opt/indico/etc
 
 COPY indico.conf /opt/indico/etc/indico.conf
-RUN ln -s /opt/indico/etc/indico.conf /etc/indico.conf
```
Comment (ThiefMaster) [inline indico/Dockerfile]: are we setting INDICO_CONFIG somewhere? Otherwise removing this will probably result in indico not finding its config file.

Comment (meluru) [reply indico/Dockerfile]: Yes, in 3rd line of this file. That's why I removed it becasue it;s redundant

File: indico/logging.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# - All log formaters may use `%(request_id)s` to get a random ID unique to each
+#   request which makes it easier to grep for all other log entries that belong to
+#   a specific request.
+# - Setting `append_request_info` to `true` in a formatter will append various
+#   information about the current request to the log message.  This will result in
+#   a multi-line message and is only recommended for errors; usually in a handler
+#   which sends emails.
+
+version: 1
+
```
Comment (mvidalgarcia) [inline indico/logging.yaml]: Any particular reason why use double break lines?

Comment (meluru) [reply indico/logging.yaml]: Actually no, I've removed it

**Pull request merged**